### DISCUSSION TITLE
Allow creating issues from task lists in a different repo

### DISCUSSION THREAD

**Discussion by lucatpa** [community/community#5325] [Projects and Issues]
The new feature to create issues from tasks lists is extremely useful for me, in particular to implement tracking issues that are composed from several smaller tasks. When doing so, sometimes one of these tasks really belong to a different repo (a dependency basically) so it would be awesome to be able to choose which repo to use, similarly to the popup one gets when converting a Project card to an issue (or creating a new issue from an issue comment).

**Comment by WardenDrew** [comment]
Related Larger Discussion:
- https://github.com/github/feedback/discussions/4442
